### PR TITLE
feat(provider/ecs): Added vertical scaling section to deploy modal

### PR DESCRIPTION
**Pull request by BrunoCarrier**
feat(provider/ecs): Added vertical scaling section to deploy modal

@anotherchrisberry @jrsquared @christopherthielen @dkirillov

### CODE REVIEW

File: app/scripts/modules/ecs/serverGroup/configure/wizard/verticalScaling/verticalScaling.component.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<div class="container-fluid form-horizontal">
+
+  <div class="form-group">
+    <div class="col-md-5 sm-label-right">
+      Reserved compute units <help-field key="ecs.capacity.reserved.computeUnits"></help-field>
+    </div>
+    <div class="col-md-2"><input type="number"
+                                 class="form-control input-sm no-spel"
+                                 ng-pattern="basicSettingsCtrl.detailPattern"
```
Comment (anotherchrisberry) [inline app/scripts/modules/ecs/serverGroup/configure/wizard/verticalScaling/verticalScaling.component.html]: remove `ng-pattern`

File: app/scripts/modules/ecs/serverGroup/configure/wizard/verticalScaling/verticalScaling.component.html [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+                                 ng-model="$ctrl.command.computeUnits"/></div>
+  </div>
+
+  <div class="form-group">
+    <div class="col-md-5 sm-label-right">
+      Reserved Memory <help-field key="ecs.capacity.reserved.memory"></help-field>
+    </div>
+    <div class="col-md-2"><input type="number"
+                                 class="form-control input-sm no-spel"
+                                 ng-pattern="basicSettingsCtrl.detailPattern"
```
Comment (anotherchrisberry) [inline app/scripts/modules/ecs/serverGroup/configure/wizard/verticalScaling/verticalScaling.component.html]: remove

File: app/scripts/modules/ecs/serverGroup/configure/wizard/verticalScaling/verticalScaling.component.html [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+                                 name="computeUnits"
+                                 ng-model="$ctrl.command.computeUnits"/></div>
+  </div>
+
+  <div class="form-group">
+    <div class="col-md-5 sm-label-right">
+      Reserved Memory <help-field key="ecs.capacity.reserved.memory"></help-field>
```
Comment (anotherchrisberry) [inline app/scripts/modules/ecs/serverGroup/configure/wizard/verticalScaling/verticalScaling.component.html]: Inconsistent casing - `Memory` should be `memory`, or `compute units` above should be `Compute Units`

File: app/scripts/modules/ecs/serverGroup/configure/wizard/verticalScaling/verticalScaling.component.html (hunk 1 above)
Comment (BrunoCarrier) [reply app/scripts/modules/ecs/serverGroup/configure/wizard/verticalScaling/verticalScaling.component.html]: Removed

File: app/scripts/modules/ecs/serverGroup/configure/wizard/verticalScaling/verticalScaling.component.html (hunk 3 above)
Comment (BrunoCarrier) [reply app/scripts/modules/ecs/serverGroup/configure/wizard/verticalScaling/verticalScaling.component.html]: Made it `Compute Units`

File: app/scripts/modules/ecs/serverGroup/configure/wizard/verticalScaling/verticalScaling.component.html (hunk 2 above)
Comment (BrunoCarrier) [reply app/scripts/modules/ecs/serverGroup/configure/wizard/verticalScaling/verticalScaling.component.html]: Removed

**Comment by BrunoCarrier** [comment]
Regarding `Guessing there is a separate PR to add the section to the modal?` --> This PR is not yet made but will be made once the other pieces are in place.

**Pull request merged**